docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
+
+A how to guide for accessing and running applications on a kubernetes cluster.
+
+### Basic Setup
+The kubernetes cluster is accessed via the ```kubectl`` command.  Full details of using the ```kubectl``` command are described [elsewhere](docs/cli.md).  Make sure that ```kubectl``` is in your path by running
+
+```sh
+which kubectl
+```
+
+If that command returns a path to the binary, you are ready to go.  If it doesn't, you need to update your PATH to include the appropriate directory from the Kubernetes directory.
+
+### Commands
+#### run
+```sh
+kubectl run <pod-name> --image=nginx [--replicas=N] [--labels=<key>=<value>,...][--export [--external]]
```

Comment:
I'm happy to address the generator concerns, that you raise, but I firmly believe that we 100% need an interface like this.  It doesn't have anything to do with long running experience with the tool.  It 100% has to do with the user's first interaction with Kubernetes.  We need to make that as clean, crisp and easy as possible.

And the user's first "kick the tires" experience is always an imperative style "run" it's not a declarative style "generate this config to run".

I agree that the path forward is from the imperative kick the tires to declarative management needs to be as smooth as possible, but we'll never even get people to try to move if we don't give them an easy on ramp.

#1695 may have requirements, but it doesn't have much of a path forward.  I intend to start coding this asap, and this PR is the roadmap for that code.